README.md

```diff
@@ -211,4 +239,10 @@ See our
 [developer resources](https://allencellmodeling.github.io/aicsimageio/developer_resources)
 for information related to developing the code.
 
+## Citation
+
+If you find `aicsimageio` useful, please cite this repository as:
+
+> AICSImageIO Contributors (2021). AICSImageIO: Image Reading, Metadata Conversion, and Image Writing for Microscopy Images in Pure Python [Computer software]. GitHub. https://github.com/AllenCellModeling/aicsimageio
+
 _Free software: BSD-3-Clause_
```

Comment:
Perhaps we should mention here that this statement applied to the code we have written but that dependent packages may have different licences? Yes it's obvious but it makes me a little nervous not to give it lip service.